### PR TITLE
update income example and yamls

### PR DESCRIPTION
**Pull request by cliveseldon**
Update income example for latest mlserver. 

NB:

- Will need to rethink folder structure in a future work for all of scv2.
- Will need Seldon deploy example updates for demos in docs

### CODE REVIEW

File: samples/examples/income_classifier/Makefile [hunk 1]
```diff
@@ -6,22 +23,14 @@ infer.md: infer.ipynb
...
-	gsutil cp -R infer-data gs://seldon-models/scv2/examples/income
+	gsutil cp -R preprocessor gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R classifier gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R explainer gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R drift-detector gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R outlier-detector gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R infer-data gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
 
 .PHONY: install-dev
 install-dev:
```
Comment (sakoush) [inline samples/examples/income_classifier/Makefile:36]: is this still required?

File: samples/examples/income_classifier/Makefile [hunk 2]
```diff
@@ -6,22 +23,14 @@ infer.md: infer.ipynb
...
-	gsutil cp -R drift-detector gs://seldon-models/scv2/examples/income
-	gsutil cp -R outlier-detector gs://seldon-models/scv2/examples/income
-	gsutil cp -R preprocessor gs://seldon-models/scv2/examples/income
-	gsutil cp -R infer-data gs://seldon-models/scv2/examples/income
+	gsutil cp -R preprocessor gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R classifier gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R explainer gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R drift-detector gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R outlier-detector gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
+	gsutil cp -R infer-data gs://seldon-models/scv2/examples/${MLSERVER_FOLDER}/income
```
Comment (sakoush) [inline samples/examples/income_classifier/Makefile:33]: how are we creating mlserver `model-settings.json`?

File: samples/examples/income_classifier/Makefile (hunk 1 above)
Comment (cliveseldon) [reply samples/examples/income_classifier/Makefile]: will remove

File: samples/examples/income_classifier/Makefile (hunk 2 above)
Comment (cliveseldon) [reply samples/examples/income_classifier/Makefile:33]: They are fixed not generated

Comment (sakoush) [reply samples/examples/income_classifier/Makefile:33]: ok

**Review by sakoush — APPROVED** [review]

**Pull request merged**